pkg/session/bootstrap.go

```diff
@@ -1247,6 +1258,7 @@ const (
 
 	// version 242
 	//   insert `cluster_id` into the `mysql.tidb` table.
+	//   Add workload-based learning system tables
```

Comment:
v242 is introduced by https://github.com/pingcap/tidb/pull/59511 recently, better to add a temporary comment which will be reduced when WBL is ready to go. In case of the author will curious about the change